students/ScottL/session04/Mailroom_Part2.py

```diff
@@ -0,0 +1,185 @@
+
+global_donation_list = [{"Name": "Tom Petty", "Amount": 5000},
+                        {"Name": "Tom Petty", "Amount": 15000},
+                        {"Name": "Neil Diamond", "Amount": 20000},
+                        {"Name": "Neil Diamond", "Amount": 30000},
+                        {"Name": "Elton John", "Amount": 25000},
+                        {"Name": "Elton John", "Amount": 25000},
+                        {"Name": "Elton John", "Amount": 40000},
+                        {"Name": "John Lennon", "Amount": 10000},
+                        {"Name": "Mick Jagger", "Amount": 20000},
+                        {"Name": "Mick Jagger", "Amount": 10000}]
```

Comment:
I think you going to wait to start your dictionary off as a list for amounts, i.e:
```
[{"Name": "Tom Petty", "Amount": 5000},
                        {"Name": "Tom Petty", "Amount": [15000]},
                        {"Name": "Neil Diamond", "Amount": [20000]},
                        {"Name": "Neil Diamond", "Amount": [30000]},
                        {"Name": "Elton John", "Amount": [25000]},
                        {"Name": "Elton John", "Amount": [25000]},
                        {"Name": "Elton John", "Amount": [40000]},
                        {"Name": "John Lennon", "Amount": [10000]},
                        {"Name": "Mick Jagger", "Amount": [20000]},
                        {"Name": "Mick Jagger", "Amount": [10000]}]
```
It will make it much easier to sort and add values from the start